docs/core/tools/extensibility.md

```diff
@@ -136,49 +135,39 @@ The sample target's project file is included below for reference. It shows how t
 
 ### PATH-based extensibility
 PATH-based extensibility is usually used for development machines where you need a tool that conceptually covers more 
-than a single project. The main drawback of this extensions mechanism is that it is tied to the machine where the 
+than a single project. The main drawback of this extension mechanism is that it's tied to the machine where the 
 tool exists. If you need it on another machine, you would have to deploy it.
 
 This pattern of CLI toolset extensibility is very simple. As covered in the [.NET Core CLI overview](index.md), `dotnet` driver 
-can run any command that is named after the `dotnet-<command>` convention. The default resolution logic will first 
-probe several locations and will finally fall to the system PATH. If the requested command exists in the system PATH 
+can run any command that is named after the `dotnet-<command>` convention. The default resolution logic first 
+probes several locations and finally falls to the system PATH. If the requested command exists in the system PATH 
 and is a binary that can be invoked, `dotnet` driver will invoke it. 
 
 The binary can be pretty much anything that the operating system can execute. On Unix systems, this means anything that 
-has the execute bit set via `chmod +x`. On Windows it means anything that Windows knows how to run. 
+has the execute bit set via `chmod +x`. On Windows, you can use *cmd* files. 
 
-As an example, let's take a look at a very simple implementation of a `dotnet clean` command. We will use `bash` to 
-implement this command. The command will simply delete the `bin/` and `obj/` directories in the current directory. If 
-the `--lock` argument is passed to it, it will also delete `project.lock.json` file. The entirety of the command is 
-given below. 
+Let's take a look at a very simple implementation of a `dotnet clean` command. We'll use `bash` to 
+implement this command. The following command will simply delete the `bin/` and `obj/` directories from the current directory.
 
 ```bash
 #!/bin/bash
 
 # Delete the bin and obj dirs
 rm -rf bin/ obj/
 
-LOCK_FILE=$1
-if [[ "$LOCK_FILE" = "--lock" ]]; then
-    rm project.lock.json
-fi
-
-
 echo "Cleaning complete..."
 ```
 
+> [!NOTE]
+> You should use `dotnet clean` in production code and not the previous command. It's only given as an example. 
+
 On macOS, we can save this script as `dotnet-clean` and set its executable bit with `chmod +x dotnet-clean`. We can then 
```

Comment:
We should most likely change this example into a simple "Hello World". This would, I believe, also alleviate the need for a Windows example, though I'm not opposed to putting it here; it just seems like filler material.